### PR TITLE
Add description to input files option

### PR DESCRIPTION
**Pull request by khou2020**
Add additional desceiption to -i option
Update README file for the new help message

### CODE REVIEW

**Comment by wkliao** [comment]
Why just folder name, but not file name?

**Comment by khou2020** [comment]
> Why just folder name, but not file name?

Like the output file, the file name is determined by the test case. Users can only pick directory name.
If a test runs both H0 and H1, there will be 2 files to read.

**Comment by wkliao** [comment]
How do users use this option? How are the file names determined?
Need to explain in the README and give an example.

**Comment by khou2020** [comment]
> How do users use this option? How are the file names determined?
> Need to explain in the README and give an example.

Fixed

File: README.md [hunk 1]
```diff
@@ -240,6 +251,11 @@ sharing Decomposition 4, 2 sharing Decomposition 5, and 4 sharing Decomposition
        [-g num] Number of IO groups (subfiles) (default 1)
        [-o output_dir] Output directory name (default ./)
        [-i target_dir] Path to directory containing the input files
+              If set, E3SM I/O will read the data from the input files and write it out in the write test.
+              The input files must be generated by E3SM under the same configuration (decomposition file) as the test case and be written by the same underlying I/O library.
```
Comment (wkliao) [inline README.md:255]: `The input files must be named the same as the output file`
Does this mean the input file will be overwritten by the output files?
Why not just different input and output file names?

Comment (khou2020) [reply README.md:255]: They are in the different folder so it won't be overwritten.
The output file name cannot be set by the user, the user can only set the folder that contains them.
We assumes the same for input files.

Comment (wkliao) [reply README.md:255]: Let's change that to let users to specify input and output file names.
For ADIOS blob method the output file name is a folder.
We can also set default values.

Comment (khou2020) [reply README.md:255]: It should be a different commit as this is about updating the descriptions.
What options do we want to use? There can be more than 1 file.

Comment (wkliao) [reply README.md:255]: In that case, create a new PR for that and I will merge that before this.

-i path:  path to input file or folder
Similar for -o.
We will describe their different usages when different API/strategy is used.

Comment (khou2020) [reply README.md:255]: F case have 2 files. How will user assign them?

Comment (wkliao) [reply README.md:255]: Append 'h0' and 'h1' but before '.nc'.

**Pull request closed without merging**